scripts/generators/es_template.py

```diff
@@ -64,6 +64,9 @@ def entry_for(field):
             field_entry['fields'] = {}
             for mf in field['multi_fields']:
                 mf_entry = {'type': mf['type']}
+                mf_normalizer = mf.get('normalizer')
+                if mf_normalizer:
+                    mf_entry['normalizer'] = mf_normalizer
```

Comment:
```suggestion
                if mf['type'] == 'keyword':
                    ecs_helpers.dict_copy_existing_keys(mf, mf_entry, ['normalizer', 'ignore_above'])
```

`ecs_helpers.dict_copy_existing_keys` allows copying in `ignore_above` along with the added `normalizer` setting and will also check if the key exists in the source dict before trying to copy.